packages/shared/src/common/enums/api/routes.ts

```diff
@@ -10,6 +10,7 @@ export enum ApiRoutes {
   VERIFY = '/verify',
   HEALTH = '/health',
   CREATE_POST = '/post',
+  BIDS = '/bids',
```

Comment:
Maybe it make sense to have items/place-bid

Reply:
But maybe it will be separate entity - Bid?